commit f70a26aaf836da0f9e242945151b49190d682d95
Author: Allen Winter <allen.winter@kdab.com>
Date:   Sat Dec 3 11:11:48 2016 -0500

    CMakeLists.txt, libical.pc.in - ICU must appear as Requires in pkgconfig
    issue#244

diff --git a/CMakeLists.txt b/CMakeLists.txt
index b3d49d0d..0f6dfc34 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -145,7 +145,7 @@ endif()
 #  RSCALE info at http://tools.ietf.org/html/rfc7529
 find_package(ICU)
 if(ICU_FOUND)
-  set(ICUUC_LIBS "-licuuc") #for libical.pc
+  set(REQUIRES_PRIVATE_ICU "Requires.private: icu-i18n") #for libical.pc
   set(HAVE_LIBICU 1)
   if(ICU_MAJOR_VERSION VERSION_GREATER 50)
     set(HAVE_ICU_DANGI TRUE)
@@ -155,7 +155,6 @@ if(ICU_FOUND)
 endif()
 if(ICU_I18N_FOUND)
   set(HAVE_LIBICU_I18N 1)
-  set(ICUI18N_LIBS "-licui18n") #for libical.pc
 endif()
 
 # MSVC specific definitions
diff --git a/libical.pc.in b/libical.pc.in
index 8899f430..c07eb52a 100644
--- a/libical.pc.in
+++ b/libical.pc.in
@@ -2,11 +2,11 @@ prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
-threadslib=@PTHREAD_LIBS@
-iculibs=@ICUUC_LIBS@ @ICUI18N_LIBS@
 
 Name: libical
 Description: An implementation of basic iCAL protocols
 Version: @LIBICAL_LIB_VERSION_STRING@
-Libs: -L${libdir} -lical -licalss -licalvcal ${threadslib} ${iculibs}
+Libs: -L${libdir} -lical -licalss -licalvcal
+Libs.private: @PTHREAD_LIBS@
+@REQUIRES_PRIVATE_ICU@
 Cflags: -I${includedir}
